packages/relay/src/ReputationManager.ts

```diff
@@ -105,7 +105,7 @@ export class ReputationManager {
     if (-changeInAbuseWindow >= this.config.abuseReputationChange) {
       await this.reputationStoreManager.setAbuseFlag(paymaster, eventBlockNumber)
     }
-    const oldChangesExpirationBlock = eventBlockNumber - this.config.abuseTimeWindowBlocks
+    const oldChangesExpirationBlock = Math.max(eventBlockNumber - this.config.abuseTimeWindowBlocks, 0)
```

Comment:
How did this change end up in this PR? Why did you have to fix the semi-abandoned reputation manager?